content/onboarding.rst

```diff
@@ -129,10 +127,10 @@ The initial stage is over when all the Onboarding tickets from the milestone ded
 
 We are far from single-sign-on and while working on this project you will have many different accounts.
 
-For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations (Chevah and ProAtria) and our core team. Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
+For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations and our core team. Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
 
 We use Skype for phone calls.
-Use the dedicated ProAtria Skype account as the official tools to collaborate with the team and to make phone calls.
+Use the dedicated Skype account as the official tools to collaborate with the team and to make phone calls.
```

Comment:
it might be a good time to give some arguments why Skype :)

Ex, It works on any platform and on low connections... any other solution  failed.

Reply:
Well no, because we want people to use other solutions if it fits the situation. For example, there are better solutions for screen sharing that includes tools.  But I think we just default to Skype.